examples/load-gen.ts

```diff
@@ -383,17 +413,21 @@ const loadGeneratorOptions: BasicJavaScriptLoadGenOptions = {
    */
   loggerOptions: {
     /**
-     * Available log levels are TRACE, DEBUG, INFO, WARN, and ERROR.  DEBUG
+     * Available log levels are TRACE, DEBUG, INFO, WARN, and ERROR.  INFO
      * is a reasonable choice for this load generator program.
      */
-    level: LogLevel.DEBUG,
+    level: LogLevel.INFO,
     /**
      * Allows you to choose between formatting your log output as JSON (a good
      * choice for production environments) or CONSOLE (a better choice for
      * development environments).
      */
     format: LogFormat.CONSOLE,
   },
+  /** Print some statistics about throughput and latency every time this many
+   *  milliseconds have passed. Default is 5 seconds.
+   */
+  printStatsEveryNMs: 5 * 1000,
```

Comment:
nit: I don't think anyone would ever want to print these more than once per second, so maybe we change it from `Ms` to `Seconds`?  not a huge deal.

Reply:
We already had `requestTimeoutMs`. 🤷 Either way is fine, if they want it every 500 ms they can set it to .5. `printStatsEveryNSeconds` is less awkward.